kubernetes/redis/statefulset-redis.yaml

```diff
@@ -22,7 +22,7 @@ spec:
           - |
             echo "Copying configuration file"
             cp /tmp/redis/redis.conf /etc/redis/redis.conf            
-            if [ "$(redis-cli -h sentinel -p 5000 ping)" != "PONG" ]; then
+            if [ "$(redis-cli --tls --cacert /certs/ca.crt -h sentinel -p 5000 ping)" != "PONG" ]; then
```

Comment:
issue will probably crop up here if forcing x509, which can deal with when can test in future